### ISSUE TITLE
Windows ignores the effects

### ISSUE THREAD

**Issue by slobodaapl** [seagull/disable-scheduledWaking#11]
Hey there, I used your tools on latest W11 builds but to no avail. The tool shows it succeeds, but in the task scheduler I can still see the task is enabled. I also tried PS utils but even then it is still enabled.

Any ideas on how to get around this?

**Comment by seagull** [comment]
What happens if you re-run the tool? Are the same tasks shown as capable of
waking the device?

**Comment by slobodaapl** [comment]
Yes it seems it is still enabled when I re-run the tool. I re-ran it once more after disabling it again and now it no longer shows up, but I wonder what triggers it to get re-enabled. I will try to figure out when this happens.
